src/docs/asciidoc/common/cloud-services/standard-user-workflow/presentation.adoc

```diff
@@ -0,0 +1,30 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+In a software application, we (almost) always need to manage users. The **Standard User Workflow** helps you to do this, in the most common use case. Currently, it only handles the registration of a user. In the following versions, this cloud service will help you to : 
+
+* Log in a user 
+* Log out a user
+* Manage a forgotten password
+* Manage permissions of users
+* etc... 
+
+NOTE: At this moment, the connection and the disconnection of a user is done through the ZetaPush SDK (We will see below how to do this).
+
+For us, the most common use case is when a user can create an account using a login and a password and needs to confirm his account via a link in an email.
+
+
+**Sign up : **
+
+image::./schemas/standardworkflowsignup.png[]
+<1> The user creates his account using a login, a password and an email. When the user submit the form, the account is created on the ZetaPush platform but his state is _waiting confirmation_.
```

Comment:
submit**s**

The last sentence is too negative and technical (meaning internal behavior) I think. 
~~the account is created on the ZetaPush platform but his state is _waiting confirmation_~~
the account is created on the ZetaPush platform. In order to ensure that the user is not a bot, the account has to be confirmed before the user is authorized to login.